### PR TITLE
Automatic user-frendly payload printing

### PR DESCRIPTION
**Pull request by sw-dev-code**

### CODE REVIEW

File: lookup_tables/VoBoTC_analog_sensor_numbers.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Analog Sensor Number,Analog Sensor Name
+0,Battery Voltge
```
Comment (tvolpe-vb) [inline lookup_tables/VoBoTC_analog_sensor_numbers.txt:2]: can you fix this misspelling of Voltage

File: vobodec.js [hunk 2]
```diff
@@ -6,24 +6,24 @@
...
-        console.log("\n\n");
-        console.log(obj.time);
-        console.log(obj.deveui);
+    if (obj.size != 0) 
+    {
+        var base64DecodedData = Buffer.from(obj.data, "base64");
+        bytes = Buffer.from(base64DecodedData, "hex");
 
-        var decodedData = Buffer.from(obj.data, "base64");
+		var payload = utils.processReceivedObject(obj);
```
Comment (tvolpe-vb) [inline vobodec.js:25]: fix extra indention here and on the next line.

File: lookup_tables/VoBoTC_analog_sensor_numbers.txt (hunk 1 above)
Comment (sw-dev-code) [reply lookup_tables/VoBoTC_analog_sensor_numbers.txt]: Done in latest commit

File: vobodec.js (hunk 2 above)
Comment (sw-dev-code) [reply vobodec.js]: Done in latest commit

**Pull request merged**